aviary/models/multi_engine_single_aisle/multi_engine_single_aisle_data.py

```diff
@@ -175,6 +175,7 @@
 engine_1_inputs.set_val(Aircraft.Engine.FLIGHT_IDLE_MIN_FRACTION, 0.08)
 engine_1_inputs.set_val(Aircraft.Engine.GEOPOTENTIAL_ALT, False)
 engine_1_inputs.set_val(Aircraft.Engine.INTERPOLATION_METHOD, 'slinear')
+engine_1_inputs.set_val(Mission.Summary.FUEL_FLOW_SCALER, 1.0)
```

Comment:
should we be adding this in check_and_preprocess_inputs (or something similar), since it seems reasonable that if a user didn't specify it, it should just be 1?

Reply:
Yeah, this should probably have a default value of 1.0 that gets set if it is missing.

Reply:
I added logic to see if that variable is in `aviary_options` and if not to set it to 1.0